CodeEdit/InspectorSidebar/InspectorSidebar.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  InspectorSidebar.swift
+//  CodeEdit
+//
+//  Created by Austin Condiff on 3/21/22.
+//
+
+import SwiftUI
+import WorkspaceClient
+
+struct InspectorSidebar: View {
+    @ObservedObject var workspace: WorkspaceDocument
+    var windowController: NSWindowController
+    @State private var selection: Int = 0
```

Comment:
I copied what was done in the navigator sidebar. I do agree though. Although it might end up having to be somewhat dynamic as we allow extensions to add new sidebar tabs.